Product helpwave impulse (#80)
helpwave/helpwave: products/impulse/README.md

# helpwave impulse

This document includes all the necessary information to understand the idea and concepts behind helpwave impulse. 

## The idea

> The aim of helpwave impulse is to reduce the load on the health care sector by helping people to live a healthier lifestyle. This will reduce or delay the occurrence of illness, which leads to a more sustainable population. Healthier in this context means an improvement of the nutrition, fitness or mental health.

Reviewer: "delay" has a weird connotation in this context. Just "reduce" should be fine.

#### Fitness

The physical activity level (PAL[^PAL]) should be used to evaluate the activity level of a user. By knowing this a suitable baseline of activity could be inferred for a user.

| PAL-Value | activity level                                           | example                                          |
|-----------|----------------------------------------------------------|--------------------------------------------------|
| 1,2 - 1,3 | sedentary or lying lifestyle                             | Immobile, bedridden people                       |
| 1,4 - 1,5 | sedentary work, little strenuous leisure time            | Office worker                                    |
| 1,6 - 1,7 | Sitting, short-term walking and standing work            | student                                          |
| 1,8 - 1,9 | Mostly walking and standing work                         | Salesperson, waiter, craftsman                   |
| 2,0 - 2,4 | Physically demanding job or very active leisure activity | Construction worker, farmer, competitive athlete |

#### Mental health

> Disclaimer: This app will not replace a professional psychologist.

> The app should evaluate the mental condition of a user. This is important so that the app isn’t overwhelming the user with the baseline tasks. 

Reviewer: @aegis301

> The app should evaluate the mental condition of a user. This is important so that the app isn’t overwhelming the user with the baseline tasks. 

Author: @MaxSchaefer I would recommend that we focus on fitness and nutrition baseline tasks for the beginning and only have a concept that doesn’t overwhelm the user. For the second stage we should seek help from a professional to get recommendation on how to design task to support mental health.

Every baseline task should have information why it was selected for a user. There should also be information displayed on why this task is imported, so that there is a possible learning from the app.

### Challenge Tasks

Challenge Tasks are Tasks that are used motivate users to keep going and bring some variety to the app. These Challenges could include nutrition or fitness tasks. These challenges should also fit the lifestyle profile of a user.

### Teams

There should be the possibility to create teams for individual users. A user should also be part of implicit teams that are based on the location. Examples are city-based teams. 

#### Company

Companies should have the ability to create company teams for their business health management. It is import that now information of a user is visible to the company itself. These teams should be used for extra challenges or activities that are company related.

Another extension to this is management of company organized sports courses. This for example includes the displaying of all necessary information for the course or the registration of a user.

### Achievements

> To add some gamification aspects to the app to let the user keep going the app should have an individual score with virtual achievements. This means that every task that is complete will give the user a certain number of points. The baseline tasks should give more points because they are more important than the challenges and the points from challenges should be caped to a maximum per day. As a result, no one should be encouraged to prefer the challenges over the baseline task in respect to the points gained. 

Reviewer: ```suggestion
To add some gamification aspects to the app to let the user keep going the app should have an individual score with virtual achievements. For every completed task, the user is gaining a more points, based on the task. The baseline tasks should give more points because they are more important than the challenges and the points from challenges should be caped to a maximum per day. As a result, no one should be encouraged to prefer the challenges over the baseline task in respect to the points gained. 
```

> There are two different achievement types. The first one is the level which corresponds to a certain score. The needed score for an achievement should raise linear. The other achievement type is for certain challenges. 

Reviewer: Please elaborate the second achievement type.

### Rewards

> Rewards are things that a user could win from a third party by matching certain criteria. Criteria can be the score, location or city of a user, a company team or completing special challenges. The most useful thing would be some sort of digital coupons.

Reviewer: ```suggestion
Rewards are some sort of physical or digital goods from third-parties of the app. Most likely, the rewards are in the context of a location/city of the user. If a challenge is part of a company, the company should be able to define there own reward.
```

# helpwave impulse

This document includes all the necessary information to understand the idea and concepts behind helpwave impulse. 

## The idea

> The aim of helpwave impulse is to reduce the load on the health care sector by helping people to live a healthier lifestyle. This will reduce the occurrence of illness, which leads to a more sustainable population. Healthier in this context means an improvement of the nutrition, fitness or mental health.

Reviewer 2: "this will reduce the burden of disease, which leads to increased public health" - better wording in the medical context. Population is usually reserved for study populations or animal populations.

## The concept

### Understanding the individual needs

> A short Questionnaire should used to evaluate the needs of a healthier lifestyle for a user. This is import because each user has an individual lifestyle which creates the problem to choose the right task to great a healthier baseline. The Questions should evaluate the needs in the dimension’s nutrition, fitness and mental health. A user should not be encouraged to appropriate bad behaviour related to their lifestyle with this app.

Reviewer 2: A short Questionnaire is used to evaluate the needs of a healthier lifestyle for a user. This is important because each user has an individual lifestyle which creates the problem to choose the right task to increase personal health. The Questions should evaluate the needs in the dimensions nutrition, fitness and mental health. A user should not be encouraged to pick up unhealthy behaviour related to their lifestyle with this app.

| PAL-Value | activity level                                           | example                                          |
|-----------|----------------------------------------------------------|--------------------------------------------------|
| 1,2 - 1,3 | sedentary or lying lifestyle                             | Immobile, bedridden people                       |
| 1,4 - 1,5 | sedentary work, little strenuous leisure time            | Office worker                                    |
| 1,6 - 1,7 | Sitting, short-term walking and standing work            | student                                          |
| 1,8 - 1,9 | Mostly walking and standing work                         | Salesperson, waiter, craftsman                   |
| 2,0 - 2,4 | Physically demanding job or very active leisure activity | Construction worker, farmer, competitive athlete |

#### Mental health

> Disclaimer: This app will not replace a professional psychologist.

The app should evaluate the mental condition of a user. This is important so that the app isn’t overwhelming the user with the baseline tasks. 

> ### Baseline Tasks (Daily's)

Reviewer 2: Dailys

> The baseline tasks are tasks that should be done every day to improve the lifestyle of a user. These tasks should be derived from the information which is given be the questionnaire. So that only relevant task that would improve the lifestyle would be chosen.

Reviewer 2: The baseline tasks are tasks that should be done every day to improve the lifestyle of a user. These tasks should be derived from the information which is given be the questionnaire. So that only relevant tasks will be chosen for the user.
